edgedb/con_utils.py

```diff
@@ -104,11 +107,24 @@ def _parse_hostlist(hostlist, port):
 def _parse_connect_dsn_and_args(*, dsn, host, port, user,
                                 password, database, admin,
                                 connect_timeout, server_settings):
-
-    if dsn:
+    if admin:
+        warnings.warn(
+            'The "admin=True" parameter is deprecated and is scheduled to be '
+            'removed. Admin socket should never be used in applications. '
+            'Use command-line tool `edgedb` to setup proper credentials.',
+            DeprecationWarning, 4)
+
+    if dsn and dsn.startswith(("edgedb://", "edgedbadmin://")):
```

Comment:
Hm, this is first time I see `edgedbadmin://` -- have we agreed to add it just yet?

Reply:
Ah, I see, it's being deprecated. nvm